data/campaigns/Eastern_Invasion/utils/abilities.cfg

```diff
@@ -527,3 +527,29 @@ When an enemy unit of lower level within a 2 hex radius engages in combat, its a
         apply_to=fire,arcane
     [/resistance]
 #enddef
+
+#textdomain wesnoth-help
+
+#define WEAPON_SPECIAL_ARCANE_EI
+    [damage_type]
+        id=arcane_damage
+        name= _ "arcane"
+        description= _ "This attack combines the arcane type with the type of weapon used so that resistance to the arcane type does not penalize the user."
+        special_note=_ "This unit can use the arcane type when the opponent is particularly sensitive to it in relation to the weapon on which it is applied."
+        alternative_type=arcane
```

Comment:
Please put a `#textdomain wesnoth-ei` at before this line, so that any future additions at the end of the file don't end up in an unexpected text domain.

Also, please put the `#textdomain wesnoth-help` immediately before `_ "arcane"`, again so that as little as possible is in an unexpected domain.

Reply:
i done that but for tests, i code what i use seem be the one who work, let me with athat, please.